accounts/views.py

```diff
@@ -6,12 +7,17 @@
 from django.contrib.auth import authenticate, login
 
 
-class RegistrationView(FormView):
+class RegistrationView(AccessMixin, FormView):
```

Comment:
Seems much better, Also use it on login view. And use LoginRequiredMixin on logout view.

Reply:
done